components/Layout.js

```diff
@@ -0,0 +1,10 @@
+import Header from "./Header";
+
+const Layout = ({ children }) => (
+  <>
+    <Header />
```

Comment:
Good choice to make the header global! Having a way go back to the home page while on the login page makes a lot of sense